### PR TITLE
Refresh /internet-of-things

### PR DESCRIPTION
**Pull request by petesfrench**
**Note: Waiting on images for the 'Our products' section, requested in the doc. can go live without**

## Done

- Refreshes /internet-of-things based on the [copydoc](https://docs.google.com/document/d/1nBx1SP3Y10gXX0rZPDaz1Gudy-hWP4XRMUcETyPBldI/edit)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- [List additional steps to QA the new features or prove the bug has been resolved]

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-6519

### CODE REVIEW

File: templates/internet-of-things/index.html [hunk 1]
```diff
@@ -25,297 +27,232 @@ <h1>Ubuntu for the Internet&nbsp;of&nbsp;Things</h1>
...
       </div>
     </div>
-    <div class="col-6 p-tile">
-      <div class="row p-tile__row">
-        <div class="col-3 col-medium-3 u-hide--small">
-          <div class="u-align--center">
-            {{
-              image (
+  </div>
+  <div class="p-section u-no-padding--bottom">
```
Comment (mtruj013) [inline templates/internet-of-things/index.html:156]: `p-section` only adds bottom padding so these can be omitted altogether to get the same spacing

File: templates/internet-of-things/index.html [hunk 2]
```diff
@@ -25,297 +27,232 @@ <h1>Ubuntu for the Internet&nbsp;of&nbsp;Things</h1>
...
         <div class="p-logo-section__item">
-          {{
-          image(
-            url="https://assets.ubuntu.com/v1/5ddba83a-logo-dell.svg",
-            alt="Dell",
-            width="88",
-            height="88",
+          {{ image (
+            url="https://assets.ubuntu.com/v1/0066964b-rigado-logo.png",
+            alt="",
```
Comment (mtruj013) [inline templates/internet-of-things/index.html:214]: Missing alt tags for these

File: templates/internet-of-things/index.html [hunk 3]
```diff
@@ -324,216 +261,231 @@ <h2>Ready&ndash;to&ndash;go gateway solutions</h2>
...
-      <h3 class="p-card__title">Xilinx Evaluation Kits and SOMs</h3>
-      <p class="p-card__content">Zynq&reg; UltraScale+&trade; MPSoCs integrate a quad-core 64-bit Arm application processor with real-time processors and FPGA logic to offer an embedded, low-latency acceleration platform.</p>
-      <p class="p-card__content"><a href="https://www.xilinx.com/applications/industrial/robotics.html#solutionStack?utm_source=canonical&utm_medium=referral&utm_campaign=ubuntu&utm_content=iot&utm_term=eval_kit">Learn more on xilinx.com&nbsp;&rsaquo;</a></p>
+    <div class="col-6">
+      <div class="p-logo-section has-misaligned-images">
+        <div class="p-logo-section__items u-align--center">
+          <div class="p-logo-section__item">
+            {{ image (
+              url="https://assets.ubuntu.com/v1/6079d02b-amd-logo.png",
+              alt="",
```
Comment (mtruj013) [inline templates/internet-of-things/index.html:350]: Same here

File: templates/internet-of-things/index.html [hunk 4]
```diff
@@ -324,216 +261,231 @@ <h2>Ready&ndash;to&ndash;go gateway solutions</h2>
...
   <div class="row">
-    <div class="col-7">
-      <h2>Best for developers</h2>
-      <p>With Ubuntu, you can develop a packaged application on your laptop, then monetise it by publishing it directly to the snap store.</p>
-      <p><a href="https://docs.ubuntu.com/core/en/">Learn about developing your IoT project with Ubuntu&nbsp;&rsaquo;</a></p>
+    <div class="col-6">
+      <h2>IoT professional services</h2>
+      <p>Our expert technical teams offer full enablement, customisation and development to ensure that taking your devices to market is fast, scalable and seamless. You can focus on your applications, we handle the rest, including full support and technical workshops.</p>
+      <p>
+        <a href="/core/services" class="p-button--positive">Learn more </a>
```
Comment (mtruj013) [inline templates/internet-of-things/index.html:457]: Left a comment in the doc asking the pdm to make this more descriptive, Ant flagged this in another recent build

**Review by mtruj013 — APPROVED** [review]
LGTM! Just some minor comments in the code

File: templates/internet-of-things/index.html (hunk 3 above)
Comment (petesfrench) [reply templates/internet-of-things/index.html:350]: I think due to the complexity of the image and lack of imformation it adds, we can omit the alt text on this one

Comment (mtruj013) [reply templates/internet-of-things/index.html:348]: This is a logo though, we always add alt text for the logos https://accessibleweb.com/question-answer/logos-need-follow-accessibility-guidelines/

Comment (petesfrench) [reply templates/internet-of-things/index.html:348]: Sorry, I thought you were referring to a different image

**Pull request merged**